sim/rogue/items_sets_pve.go

```diff
@@ -156,8 +157,9 @@ var ItemSetNightSlayerThrill = core.NewItemSet(core.ItemSet{
 				OnExpire: func(aura *core.Aura, sim *core.Simulation) {
 					core.Each(affectedSpells, func(spell *core.Spell) { spell.Cost.Multiplier += 100 })
 				},
+				//Poisoned Knife is ignored
 				OnCastComplete: func(aura *core.Aura, sim *core.Simulation, spell *core.Spell) {
-					if aura.RemainingDuration(sim) == aura.Duration || spell.DefaultCast.Cost == 0 {
+					if aura.RemainingDuration(sim) == aura.Duration || spell.DefaultCast.Cost == 0 || spell.ActionID.SpellID == 425012 {
```

Comment:
Are you sure that Poisoned Knife isn't included? I see it in the affected spells on wowhead
https://www.wowhead.com/classic/spell=467735/clearcasting
<img width="592" alt="image" src="https://github.com/user-attachments/assets/e180e36b-e7e4-4254-99a5-25647c4a827e">

Reply:
It is affected but does not consume.  Don't rat me out again haha!

Reply:
I mean 😅